code/modules/admin/admin_ranks.dm

```diff
@@ -58,6 +58,28 @@ var/list/admin_ranks = list()								//list of all ranks with associated rights
 	load_admins()
 	return 1
 
+/proc/parse_admins_config_file()
+	var/list/admins = new/list()
+	for(var/line in file2list("config/admins.txt"))
+		// strip out comments
+		var/comment_start = findtext(line, "#")
+		if(comment_start > 0)
+			line = copytext(line, comment_start)

```

Comment:
idea of the comments is to ignore that line. Not to remove the # of the line then continue

Reply:
The intention was to remove everything after `#`, not everything before... Ooops...
Fixed now, thanks!